HYUNSANG_OH/app.js

```diff
@@ -4,9 +4,14 @@ const cors = require("cors");
 const morgan = require("morgan");
 const dotenv = require("dotenv");
 const { DataSource } = require("typeorm");
-
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
 dotenv.config();
 
+//암호화 관련 코드
+const secretKey = process.env.secretKey;
+const saltRounds = 12;
```

Comment:
```suggestion
const saltRounds = process.env..;
```
개인적으로 이것도 환경변수로 관리하면 더 좋을 것 같아요~